src/main/java/org/terasology/namegenerator/creature/CreatureAssetTheme.java

```diff
@@ -16,13 +16,13 @@
 
 package org.terasology.namegenerator.creature;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.terasology.asset.Assets;
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.namegenerator.data.NameGeneratorComponent;
 
+import java.util.Collections;
```

Comment:
I think that it's a good idea, though. Can you open an issue at Terasology or it? I haven't managed to figure out *how* eclipse does the sorting, but it can import/export sorting configurations. Maybe that's compatible to IntelliJ? Needs further investigation (if pursued) in any case.